lib/bigtable/table.js

```diff
@@ -0,0 +1,835 @@
+  }
+
+  if (options.limit) {
+    reqOpts.numRowsLimit = options.limit;
+  }
+
+  var stream = through.obj();
+
+  this.request(grpcOpts, reqOpts)
+    .on('error', stream.emit.bind(stream, 'error'))
+    .on('end', stream.end.bind(stream))
+    .on('data', function(rowData) {
+      var row = self.row(rowData.rowKey);
+      row.cells = Row.formatChunks_(rowData.chunks);
+      stream.emit('data', row);
+    });
+
+  if (!is.function(callback)) {
+    return stream;
+  }
+
+  stream
+    .on('error', callback)
+    .pipe(concat(function(rows) {
+      callback(null, rows);
+    }));
+};
+
+/**
+ * Insert or update rows in your table.
+ *
+ * @param {object|object[]} entries - List of entries to be inserted.
+ *     See {module:bigtable/table#mutate}
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * var entries = [{
+ *   key: 'lincoln', // row key
+ *   data: {
+ *     name: { // family name
+ *       first: 'Abraham', // column name + value
+ *       last: 'Lincoln'
+ *     }
+ *   }
+ * }];
+ *
+ * table.insert(entries, function(err, apiResponse) {});
+ */
+Table.prototype.insert = function(entries, callback) {
+  entries = arrify(entries)
+    .map(propAssign('method', Entry.methods.INSERT));
+
+  return this.mutate(entries, callback);
+};
+
+/**
+ * Apply a set of changes to be atomically applied to the specified row(s).
+ * Entries are applied in order, meaning that earlier mutations can be masked
+ * by later ones.
+ *
+ * @param {object|object[]} entries - List of entries to be inserted or deleted.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * //-
+ * // Insert an entry. See {module:bigtable/table#insert}
+ * //-
+ * var callback = function(err, apiResponse) {
+ *   if (!err) {
+ *     // Mutation was successful.
+ *   }
+ * };
+ *
+ * var entry = {
+ *   method: 'insert',
+ *   key: 'kennedy',
+ *   data: {
+ *     name: {
+ *       first: 'John',
+ *       last: 'Kennedy'
+ *     }
+ *   }
+ * };
+ *
+ * table.mutate(entry, callback);
+ *
+ *
+ * //-
+ * // Delete an entry. See {module:bigtable/row#deleteColumns}
+ * //-
+ * var entry = {
+ *   method: 'delete',
+ *   key: 'kennedy',
+ *   data: {
+ *     family: 'name'
+ *   }
+ * };
+ *
+ * table.mutate(entry, callback);
+ *
+ * //-
+ * // Or make multiple mutations. This must contain at least one entry and at
+ * // most 100000.
+ * //-
+ * var entries = [{
+ *   method: 'insert',
+ *   key: 'lincoln',
+ *   data: {
+ *     name: {
+ *       first: 'Abraham',
+ *       last: 'Lincoln'
+ *     }
+ *   }
+ * }, {
+ *   method: 'delete',
+ *   key: 'lincoln',
+ *   data: {
+ *     family: 'name'
+ *   }
+ * }];
```

Comment:
Our style throughout the lib has been "fully expanded":

```js
var entries = [
  {
    method: 'insert'
    // ...
```

 It doesn't bug me too much if it seeps into our own code, but for the docs, it would be 💯 to be consistent.